### ISSUE TITLE
Missing curl dependency?

### ISSUE THREAD

**Issue by revodavid** [wesm/feather#145]
Noticed this when attempting to install on Windows using Microsoft R Open 3.2.5:

```
> devtools::install_github("wesm/feather/R")
Downloading GitHub repo wesm/feather@master
from URL https://api.github.com/repos/wesm/feather/zipball/master
Error in loadNamespace(name) : there is no package called ‘curl’
```

Installing curl manually fixed this particular issue.

**Comment by wesm** [comment]
curl is in suggests for devtools, I guess this should be included in the README

**Comment by dgs0323** [comment]
@revodavid I noticed you are using feather with Windows using Microsoft R Open 3.2.5 (my exact configuration). May I ask how you dealt with the C++11 gcc toolchain that is supposed to be included in R 3.3.0 [link ](https://blog.rstudio.org/2016/03/29/feather/)? I am using MRO, and 3.3.0 obviously isn't out yet. I have not been able to successfully install. 

`> devtools::install_github("wesm/feather/R")
Downloading GitHub repo wesm/feather@master
from URL https://api.github.com/repos/wesm/feather/zipball/master
Installing feather
Skipping 1 unavailable package: tibble
"C:/PROGRA~1/MIE74D~1/MRO/R-32~1.3/bin/x64/R" --no-site-file --no-environ  \
  --no-save --no-restore CMD INSTALL  \
  "C:/Users/dserfass/AppData/Local/Temp/Rtmp4QXe2r/devtools4e08436b3993/wesm-feather-84f9f91/R"  \
  --library="C:/Program Files/Microsoft/MRO/R-3.2.3/library" --install-tests 

* installing *source* package 'feather' ...

   **********************************************
   WARNING: this package has a configure script
         It probably needs manual configuration
   **********************************************


** libs
g++ -m64 -std=c++0x -I"C:/PROGRA~1/MIE74D~1/MRO/R-32~1.3/include" -DNDEBUG -I.   -I"C:/Program Files/Microsoft/MRO/R-3.2.3/library/Rcpp/include" -I"c:/applications/extsoft/include"     -O2 -Wall  -mtune=core2 -c RcppExports.cpp -o RcppExports.o
In file included from RcppExports.cpp:4:0:
feather_types.h:2:25: fatal error: feather/api.h: No such file or directory
compilation terminated.
make: *** [RcppExports.o] Error 1
Warning: running command 'make -f "Makevars" -f "C:/PROGRA~1/MIE74D~1/MRO/R-32~1.3/etc/x64/Makeconf" -f "C:/PROGRA~1/MIE74D~1/MRO/R-32~1.3/share/make/winshlib.mk" CXX='$(CXX1X) $(CXX1XSTD)' CXXFLAGS='$(CXX1XFLAGS)' CXXPICFLAGS='$(CXX1XPICFLAGS)' SHLIB_LDFLAGS='$(SHLIB_CXX1XLDFLAGS)' SHLIB_LD='$(SHLIB_CXX1XLD)' SHLIB="feather.dll" WIN=64 TCLBIN=64 OBJECTS="RcppExports.o feather-read.o feather-types.o feather-write.o"' had status 2
ERROR: compilation failed for package 'feather'
* removing 'C:/Program Files/Microsoft/MRO/R-3.2.3/library/feather'
Error: Command failed (1)`

**Comment by hadley** [comment]
I think that's a devtools bug, not a feather bug
